.github/actions/spelling/Spelling.md

```diff
@@ -0,0 +1,14 @@
+# Spell checking
+
+## Basics
+
+Since this action is an extension of [check-spelling](https://github.com/check-spelling/check-spelling), familiarize yourself with its concepts and configuration options, especially about [dictionary files](https://github.com/check-spelling/check-spelling).
+One important concept is that the spell checking process replaces matched words with a space character and built-in dictionaries are checked first. The order of entries within a dictionary file also matters.
+
+## Guidelines for adding new dictionary entries
+
+These help you decide if you should, and if yes, in what way add new entries to the configuration files, such as `allow.txt`, `expect.txt`, `patterns.txt`, etc.
```

Comment:
Use the conventional `_allow.txt_` form.

Reply:
Updated the syntax I wrote. Some of the stuff I moved was using both backtick and underscore, some just backtick, so updated those to underscore to be in accordance with https://orcharddojo.net/orchard-resources/CoreLibrary/ContributionGuideLines.